src/main/java/core/service/file/FileService.java

```diff
@@ -0,0 +1,10 @@
+package core.service.file;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface FileService {
+    String PATH_INPUT = "src/main/resources/report_input.csv";
```

Comment:
lets move this variable from interface to its implementation class OsFileServiceImpl.